src/components/Equity/EquityStakingRow.tsx

```diff
@@ -61,6 +64,52 @@ export const EquityStakingRow: React.FC<EquityStakingRowProps> = ({
 
   const asset = useAppSelector(state => selectAssetById(state, assetId))
 
+  const underlyingAssetsWithBalancesAndIcons = useAppSelector(state =>
+    opportunity?.userStakingId
+      ? selectUnderlyingStakingAssetsWithBalancesAndIcons(state, {
+          userStakingId: opportunity.userStakingId,
+        })
+      : undefined,
+  )
+
+  const assetMarketData = useAppSelector(state =>
+    selectMarketDataByAssetIdUserCurrency(state, assetId),
+  )
+
+  const { cryptoAmountPrecision, fiatAmount } = useMemo(() => {
```

Comment:
```suggestion
  const { amountCryptoPrecision, amountUserCurrency } = useMemo(() => {
```

Reply:
Probably worth updating in `EarnDefiTypeBase` as well as this is  not consistent with our latest vernacular

Reply:
Done in [4bade8f](https://github.com/shapeshift/web/pull/7644/commits/4bade8f9fe1ac3742ff2474ea6faa38eed2ad30e)

Happy to update EarnDefiTypeBase as followup to keep sanity of this PR, this is generated around 30 warnings